ragen/env/frozen_lake/utils.py

```
@@ -1,8 +1,72 @@
-import random
 import numpy as np
-from typing import List, Optional
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.14.4)

`typing.Dict` is deprecated, use `dict` instead

(UP035)